src/index.html

```diff
@@ -10,23 +10,49 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search-bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="search-bars">
+      <form
+        class="search-bar search-bar--big"
+        data-qa="big"
+      >
+        <img
+          src="./images/Search.svg"
+          alt="Search"
+          class="search-bar__icon search-bar__icon--big"
+        />
```

Comment:
Redundant, because you need to add icon with background property 
<img width="538" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/c3898359-b68b-42a8-aa37-f0f32fd5ee1d">
```suggestion
```